bench/lib/access_token.js

```diff
@@ -2,7 +2,8 @@ const accessToken = (
     process.env.MapboxAccessToken ||
     process.env.MAPBOX_ACCESS_TOKEN ||
     getURLParameter('access_token') ||
-    localStorage.getItem('accessToken')
+    localStorage.getItem('accessToken') ||
+    'pk.eyJ1IjoibWFwYm94LWdsLWpzIiwiYSI6ImNram9ybGI1ajExYjQyeGxlemppb2pwYjIifQ.LGy5UGNIsXUZdYMvfYRiAQ'
```

Comment:
nit: I would add a comment about the restriction around this access token, since it looks like it is dedicated for forked PRs.
- Who owns it, how to generate a new one if needed
- Restriction of it as described in the PR

Q: Could this access token be used for release smoke testing as well? We're currently prompting for a token in https://76953-8629417-gh.circle-artifacts.com/0/test/release/index.html, that would remove that extra step.

Reply:
I added a comment explaining why this token exists and that it's restricted to localhost. I'll document in our internal repo how to access and change the token. I don't think that should be in a publicly visible comment.

>Q: Could this access token be used for release smoke testing as well? We're currently prompting for a token in https://76953-8629417-gh.circle-artifacts.com/0/test/release/index.html, that would remove that extra step.

How is this page generated? I'd think we'd be able to use this fallback token for it.

Reply:
It's directly taken from here: https://github.com/mapbox/mapbox-gl-js/blob/main/test/release/index.js#L114. We could just override that.

Reply:
The release test page does fall back to the new token if one is not otherwise defined. I simplified that part of the code.